includes/definitions/ecreso-fm-tx.yaml

```diff
@@ -0,0 +1,28 @@
+os: worldcastsystems
+text: "Ecreso FM Transmitter"
+type: transmitter
+group: ecreso
+icon: worldcastsystems_square
```

Comment:
Make sure the images are just called the vendor name, so worldcastsystems and ensure the image files are named that.